app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,53 @@
+class ReviewsController < ApplicationController
+  # This controller is nested - its routes are under /restaurants/:restaurant_id/...
+
+  before_action :query_review, only: [:show, :update, :destroy]
+
+  def index
+    render json: Review.where(restaurant_id: params.require(:restaurant_id)).all
```

Comment:
`all` is the default scope, no need to specify it explicitly unless you really want something like `Review.all`.

May I also suggest:
```ruby
Review.find_by(restaurant_id: params.require(:restaurant_id))
```

Reply:
`find_by` will return only one review, right? (since it does `find` and then `first`) - and I want them all.